packages/polling-controller/src/PollingController.ts

```diff
@@ -86,20 +98,18 @@ function PollingControllerMixin<TBase extends Constructor>(Base: TBase) {
         throw new Error('pollingToken required');
       }
       let found = false;
-      this.#networkClientIdTokensMap.forEach((tokens, networkClientId) => {
+      this.#networkClientIdTokensMap.forEach((tokens, id) => {
         if (tokens.has(pollingToken)) {
           found = true;
-          this.#networkClientIdTokensMap
-            .get(networkClientId)
-            ?.delete(pollingToken);
-          if (this.#networkClientIdTokensMap.get(networkClientId)?.size === 0) {
-            clearTimeout(this.#intervalIds[networkClientId]);
-            delete this.#intervalIds[networkClientId];
-            this.#networkClientIdTokensMap.delete(networkClientId);
-            this.#callbacks.get(networkClientId)?.forEach((callback) => {
-              callback(networkClientId);
+          this.#networkClientIdTokensMap.get(id)?.delete(pollingToken);
```

Comment:
Don't we already have the value at `id` available as `tokens`? Otherwise, do we want to throw if a value at `id` cannot be found? It seems like we wouldn't want to do nothing.

Reply:
`tokens` here is a `set` of pollingTokens, it does not contain the `networkClientId` + `options`.